Yatima/ForLurkRepo/DSL.lean

```diff
@@ -100,7 +100,7 @@ partial def elabLurkIdents (i : TSyntax `ident) : TermElabM Expr := do
     let e ← whnf e
     let type ← inferType e
     match type.getAppFn with 
-    | .const ``List _ _ => return e
+    | .const ``List _ => return e
```

Comment:
Not sure exactly what changed, but now Lean is complaining about having even a single _.

Reply:
Wait did we revert a version? The underscore is for the data field, which was removed

Reply:
Ah ok I see, it should be ok like that